.misc/generate_dictionary.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+# generate_dictionary.py
+#
+# Generates custom documentation dictionary for SpellCheckBear
+
+import os
+import sys
+from subprocess import call
+from scspell import __file__ as scfile
+
+
+def get_data_dir(prog_name):
```

Comment:
you can use http://pypi.python.org/pypi/appdirs for this.  It will be installed as a coala dependency.